docs/buildah-build.md

```diff
@@ -1,15 +1,17 @@
-# buildah-bud "1" "April 2017" "buildah"
+# buildah-build "1" "April 2017" "buildah"
```

Comment:
You need to add a link for this so if a user
does man buildah bud, this man page comes up

Reply:
How would the link look like? Should I keep ` # buildah-bud "1" "April 2017" "buildah"` in addition to ` # buildah-build "1" "April 2017" "buildah"`

Reply:
No I am talking about something like:

https://github.com/containers/podman/tree/main/docs/source/markdown/links

Reply:
I've added extra dir for links, and modified Makefile to copy those during installation.